homeassistant/components/todo/__init__.py

```diff
@@ -269,6 +269,10 @@ async def async_delete_todo_items(self, uids: list[str]) -> None:
         """Delete an item in the To-do list."""
         raise NotImplementedError
 
+    async def async_find_todo_item(self, item: str) -> TodoItem | None:
```

Comment:
Needing to extend the entity model is unfortunate. Could it work to make this call the remove service instead? (The add method could do the same)

Reply:
fixed in https://github.com/home-assistant/core/pull/127046/commits/dec6db37f1857314d4ecc7b796052039fabaf0c7